### PR TITLE
Add flag to add DO NOT MODIFY comment in protofile

### PR DESCRIPTION
**Pull request by elyashivlavi**
In our workflow we generate the protos automatically from openapi, and
submit them together with other (manually-written) protofiles. So in
order to differentiate between the manual and the autogenerated, we
need to add a comment on top

### CODE REVIEW

File: proto_test.go [hunk 1]
```diff
@@ -2,6 +2,7 @@ package openapi2proto_test
 
 import (
 	"bytes"
+	"github.com/NYTimes/openapi2proto/protobuf"
```
Comment (jprobinson) [inline proto_test.go]: To keep with style conventions, mind grouping this import separate from the stdlib imports?

More details here: https://github.com/golang/go/wiki/CodeReviewComments#imports

Comment (elyashivlavi) [reply proto_test.go]: Thanks for the comments!
And thanks for this great project :)
I just ran `goimport`

File: proto_test.go [hunk 2]
```diff
@@ -8,18 +8,21 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/NYTimes/openapi2proto/protobuf"
+
```
Comment (jprobinson) [inline proto_test.go]: Mind grouping `protofbuf` import with its other common `openapi2proto` imports below? `goimports` can also do this for you automatically next time, if you hook it into your tooling.

Comment (elyashivlavi) [reply proto_test.go]: Weird, wonder why goimports put this space...

**Pull request merged**